src/components/App.js

```diff
@@ -0,0 +1,56 @@
+import { getUser, getFolder } from "../api";
+import "./App.css";
+import NavigationBar from "./NavigationBar";
+import { useEffect, useState } from "react";
+import FolderBar from "./FolderBar";
+import CardSection from "./CardSection";
+import Footer from "./Footer";
+
+function App() {
+  const [profile, setProfile] = useState(null);
+  const [folderName, setFolderName] = useState("");
+  const [userName, setUserName] = useState("");
+  const [profileImage, setProfileImage] = useState("");
+  const [cardList, setCardList] = useState(null);
+
+  useEffect(() => {
+    async function getProFile() {
+      const user = await getUser();
+      if (user) {
+        const { email, profileImageSource } = user;
+        setProfile({ email, profileImageSource });
+      }
+    }
+
+    getProFile();
+  }, []);
+
+  useEffect(() => {
+    async function getProFileFolder() {
```

Comment:
folder 데이터를 불러오는 것이기 때문에 간단하게 `fetchFolder`로 처리되어도 괜찮을 것 같아요~
```suggestion
    async function fetchFolder() {
```